frontend/pluginloader.lua

```diff
@@ -49,6 +49,11 @@ function PluginLoader:loadPlugins()
         else
             logger.err("extra_plugin_paths config only accepts string or table value")
         end
+    else
+        local datadir = require("datastorage"):getDataDir()
```

Comment:
Probably best to keep it data_dir just like there.

Reply:
you can also move this after the `if` so it is not called on other platforms, and then change it to just a `elseif`